### PR TITLE
remove cache

### PR DESCRIPTION
**Pull request by mixailflash**

### CODE REVIEW

**Comment by neongreen** [comment]
Can you setup Git so that it would include emails in commits? Currently the commit doesn't show up as belonging to your Github account.

File: src/Guide/Api/Methods.hs [hunk 1]
```diff
@@ -159,15 +155,3 @@ dbQuery db x = liftIO $
 
 -- Twins of corresponding functions used in "Guide.Handlers".
```
Comment (neongreen) [inline src/Guide/Api/Methods.hs]: This comment should be removed as well.

File: src/Guide/Main.hs [hunk 2]
```diff
@@ -227,7 +224,7 @@ mainWith config = do
         spc_maxRequestSize = Just (1024*1024),
         spc_csrfProtection = True,
         spc_sessionCfg = sessionCfg }
-    when (_prerender config) $ prerenderPages config db
+--    when (_prerender config) $ prerenderPages config db
```
Comment (neongreen) [inline src/Guide/Main.hs]: And this one.

File: src/Guide/Main.hs [hunk 3]
```diff
@@ -73,7 +72,7 @@ import qualified Data.Text as T
 import qualified Data.Text.IO as T
 import qualified Network.Wai.Metrics as EKG
 import qualified SlaveThread as Slave
-import qualified System.FSNotify as FSNotify
+--import qualified System.FSNotify as FSNotify
```
Comment (neongreen) [inline src/Guide/Main.hs]: You can just remove this line.

File: src/Guide/Views.hs [hunk 4]
```diff
@@ -687,6 +686,15 @@ renderSearch mbSearchQuery =
   mustache "search" $ A.object [
     "query" A..= mbSearchQuery ]
 
+
+-- temporary function
```
Comment (neongreen) [inline src/Guide/Views.hs]: Tak, a eto chto takoe

File: src/Guide/Views.hs [hunk 5]
```diff
@@ -695,32 +703,33 @@ renderSearch mbSearchQuery =
...
-               p_ $ sequence_ $ intersperse ", " $
-                 map mkCategoryLink cats
-        -- Stub categories, separated with commas
-        do let cats = filter ((== CategoryStub) . view status) catsInGroup
-           unless (null cats) $
-             div_ [class_ "categories-stub"] $ do
-               h3_ "To be written"
-               p_ $ sequence_ $ intersperse ", " $
-                 map mkCategoryLink cats
+renderCategoryList allCats = toHtmlRaw =<< (liftIO $ renderBST gen)
```
Comment (neongreen) [inline src/Guide/Views.hs]: Hm, how come just removing `cached CacheCategoryList` didn't work in this case?

File: src/Guide/Main.hs (hunk 3 above)
Comment (neongreen) [reply src/Guide/Main.hs]: And probably also remove `fsnotify` (or whatever it's called) from our cabal and stack files.

File: src/Guide/Main.hs [hunk 6]
```diff
@@ -146,8 +145,6 @@ mainWith config = do
   -- Emptying the cache is needed because during development (i.e. in REPL)
```
Comment (neongreen) [inline src/Guide/Main.hs]: Remove this comment.

File: src/Guide/Views.hs (hunk 5 above)
Comment (mixailflash) [reply src/Guide/Views.hs]: No, a problem with the list of Category

File: src/Guide/Api/Methods.hs (hunk 1 above)
Comment (mixailflash) [reply src/Guide/Api/Methods.hs]: removed

File: src/Guide/Main.hs (hunk 2 above)
Comment (mixailflash) [reply src/Guide/Main.hs]: removed

File: src/Guide/Main.hs (hunk 3 above)
Comment (mixailflash) [reply src/Guide/Main.hs]: removed line and from cabal

File: src/Guide/Views.hs (hunk 4 above)
Comment (mixailflash) [reply src/Guide/Views.hs]: removed

File: src/Guide/Main.hs (hunk 6 above)
Comment (mixailflash) [reply src/Guide/Main.hs]: removed

**Comment by willbasky** [comment]
Frontend tests hasn`t passed.

**Comment by mixailflash** [comment]
I know. But I don't know how to fix it.

**Comment by willbasky** [comment]
I restart the test! Now :green_apple:

**Pull request merged**